internal/cli/iam/teams/users/add.go

```diff
@@ -48,13 +48,13 @@ func (opts *AddOpts) Run() error {
 	return opts.Print(r)
 }
 
-// mongocli iam team(s) user(s) add username1 username2 .. usernameN --teamId teamId --orgId orgId
+// mongocli iam team(s) user(s) add userID1 userID2 .. userIDn --teamId teamId --orgId orgId
 func AddBuilder() *cobra.Command {
 	opts := &AddOpts{}
 	cmd := &cobra.Command{
-		Use:   "add username1 username2 .. usernameN",
+		Use:   "add userID1 userID2 .. userIDn",
```

Comment:
let's follow [POSIX style guide
](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html)
> Ellipses ( "..." ) are used to denote that one or more occurrences of an operand are allowed. When an option or an operand followed by ellipses is enclosed in brackets, zero or more options or operands can be specified. The form:
>utility_name [-g option_argument]...[operand...]
indicates that multiple occurrences of the option and its option-argument preceding the ellipses are valid, with semantics as indicated in the OPTIONS section of the utility. (See also Guideline 11 in Utility Syntax Guidelines .)
>The form:
>utility_name -f option_argument [-f option_argument]... [operand...]
indicates that the -f option is required to appear at least once and may appear multiple times.

This means this should be 
```suggestion
		Use:   "add <userId> [userId]...",
```

Reply:
TIL!